ui/pages/confirmations/components/confirm/info/typed-sign/permit-simulation/permit-simulation.tsx

```diff
@@ -53,6 +53,10 @@ const PermitSimulation: React.FC<{
   }, [exchangeRate, value]);
 
   const { tokenValue, tokenValueMaxPrecision } = useMemo(() => {
+    if (!value) {
+      return { tokenValue: '', tokenValueMaxPrecision: '' };
```

Comment:
If we have nothing to render, could we skip rendering the entire row and `return null`?